docs/unity/features/fps-capture.md

```diff
@@ -0,0 +1,49 @@
+---
+title: FPS Capture
+description: FPS Capture for the Embrace Unity SDK
+sidebar_position: 19
+---
+
+# Overview
+
+In mobile games and apps it is crucial to know when/if your app is having FPS issues. Typically you want a target minimum frame rate for your app and our automatic FPS capture tool will log periodically how your app is doing and if you went below a target threshold.
+
+## Getting Started
+
+Start by going to **Tools > Embrace > Settings**
+
+Under the **Settings** Tab enable:
+**Auto Instrumentation FPS Capture**
+
+Please note this is an opt-in feature.
+
+### Implementation
+
+The FPS Capture tool is automatically added to your application as soon as it starts. If you want to change your target threshold go to Embrace SDK > Resources > EmbraceFrameMeasurer.prefab and modify the `Target Frame Rate` value.
```

Comment:
>as soon as it starts

Is this technically correct? Or is it added as soon as Embrace starts in the app?